pkg/serviceaccount/jwt.go

```diff
@@ -24,6 +24,8 @@ import (
 	"io/ioutil"
 	"strings"
 
+	"github.com/golang/glog"
+
```

Comment:
nit: this and jwt-go should be in the same block, and they should be after the kubernetes imports.